src/editor/nodes/VideoNode.js

```diff
@@ -18,22 +18,44 @@ export default class VideoNode extends EditorNodeMixin(Video) {
   static async deserialize(editor, json, loadAsync, onError) {
     const node = await super.deserialize(editor, json);
 
-    const {
-      src,
-      controls,
-      autoPlay,
-      loop,
-      audioType,
-      volume,
+    const videoComp = json.components.find(c => c.name === "video");
+    const { src, controls, autoPlay, loop, projection } = videoComp.props;
+
+    let audioType,
+      gain,
       distanceModel,
       rolloffFactor,
       refDistance,
       maxDistance,
       coneInnerAngle,
       coneOuterAngle,
-      coneOuterGain,
-      projection
-    } = json.components.find(c => c.name === "video").props;
+      coneOuterGain;
+    const audioParamsComp = json.components.find(c => c.name === "audio-params");
```

Comment:
I wonder if this warrants a version bump of the spoke scene version and  "migration". See:  https://github.com/mozilla/Spoke/blob/dev/src/editor/nodes/SceneNode.js#L196 ... Not sure what our criteria are for this.

Reply:
Yep, I think think you are right, I''m going to add a migration step. It's better than adding backwards compatibility and a good practice.